pages/tutorials/cxx-guide/compiling.rst

```diff
@@ -18,6 +18,19 @@ specifying the appropriate header and library paths, and specifying the required
 libraries when linking. It is also necessary to specify the paths for libraries
 used by Cantera, such as Sundials, BLAS, and LAPACK.
 
+Instructions below assume that the environment variables ``LD_LIBRARY_PATH`` and
+``PKG_CONFIG_PATH`` are set. If they are not already set elsewhere, they can be set
+from the command line as:
+
+.. code:: bash
+
+   LD_LIBRARY_PATH=/path/to/lib
+   PKG_CONFIG_PATH=/path/to/lib/pkgconfig
+
+Note that when using *conda*, environment variables need to be specified as
+``conda env config vars set name=value`` (see `conda user guide
+<https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#setting-environment-variables>`__.
```

Comment:
I think there are couple of things getting mixed up here. To set _environment_ variables that affect subsequent commands, you need to run `export PKG_CONFIG_PATH=/path/to/lib/pkgconfig` for example. What's shown above is just defining a local variable.

The suggestion for conda here is something else -- that's a way of specifying environment variables that will then be set whenever the environment is activated. This might make an interesting addition to our conda package, if that's something that can be incorporated there, but I'm not sure how relevant it is here.

Reply:
:joy: ... yup, this explains a couple of things! This just shows how much I relied on `setup_cantera` in the past as I was pretty much able to never define any variables on the CLI. Had thought that `export` was only necessary from within a file. Oh well.

Fair point about the *conda* pathway. I can't really incorporate this here as this PR targets `main` and some of the new conda changes point to `testing`.